ctl/monitoring/monitoring.go

```diff
@@ -63,8 +63,8 @@ kmeshctl monitoring --all enable/disable`,
 		},
 	}
 	cmd.Flags().String("accesslog", "", "Control accesslog enable or disable")
-	cmd.Flags().String("all", "", "Control accesslog and metrics enable or disable together")
-	cmd.Flags().String("workloadMetrics", "", "Control Metrics for workload enable or disable")
+	cmd.Flags().String("all", "", "Control accesslog and metrics(to-service and to-pod) enable or disable together")
+	cmd.Flags().String("workloadMetrics", "", "Control metrics to connections between pods enable or disable")
```

Comment:
IIUC, when disbaled, it still record metrics with service granularity

Reply:
If `--all` is  enable, `--workloadMetrics` is disable, it  still record metrics with service granularity.